test/Microsoft.AspNet.Mvc.FunctionalTests/compiler/resources/TagHelperSample.Web.TagHelper.ConditionalComment.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link href="/css/sub/blank.css" rel="stylesheet" data-test="value" />
+    <link href="site.min.css" rel="stylesheet" data-test="value" />
+<meta name="x-stylesheet-fallback-test" content="" class="fallback-test" /><script>!function(a,b,c){var d,e=document,f=e.getElementsByTagName("SCRIPT"),g=f[f.length-1].previousElementSibling,h=e.defaultView&&e.defaultView.getComputedStyle?e.defaultView.getComputedStyle(g):g.currentStyle;if(h&&h[a]!==b)for(d=0;d<c.length;d++)e.write('<link rel="stylesheet" href="'+c[d]+'"/>')}("visibility","hidden",["\/site.css"]);</script>
```

Comment:
Nah, it's a sample. We're just trying to validate it runs and works as expected. Removing Layout would be prioritizing the testing aspects of it which is not the intent.